### PR TITLE
Update the Portuguese language

### PR DESCRIPTION
**Pull request by borfast**
* Update the language used, as requested in #21.
* Add the Github / open source block.
* Translate the disclaimer block.

### CODE REVIEW

File: site/pt.njk [hunk 1]
```diff
@@ -127,13 +127,17 @@ lang: pt
...
+                 <strong>Aviso legal:</strong> O conteúdo deste website não constitui conselho médico. A informação disponibilizada acima tenciona ser uma sugestão para indivíduos interessados seguirem um conjunto de linhas orientadoras baseadas nas melhores práticas e evidências informais para se protegerem a si e aos seus. Não há garantia ou evidência científica que as referidas linhas orientadoras levarão a uma redução no número de infecções ou que travem a pandemia da COVID-19. Também não há garantia que seguir estas linhas orientadoras reduza o risco de ser infectado com o SARS-CoV-2 ou qualquer outro contágio viral ou bacteriano.
+                 Toda a informação fornecida neste website é disponibilizada com a melhor intenção e não deverá servir como informação factual. Em caso algum serão os operadores deste website responsabilizados por qualquer reivindicação, danos ou outras obrigações. Se tem alguma preocupação ou comentário a fazer sobre a informação fornecida neste website, por favor escreva para <a class="text-gray-600 hover:text-pink-500" href="mailto:inquiry@staythefuckhome.com">inquiry@staythefuckhome.com</a>.
```
Comment (flore2003) [inline site/pt.njk]: If I'm not mistaken, there is a slight error in the translation here. The English reads `and will to provide factual information.` Your translation, if I'm not mistaken, says that website should not be taken as factual information, which is quite the opposite. Can you revise this?

**Review by flore2003 — DISMISSED** [review]
See the comment on line 140

File: site/pt.njk (hunk 1 above)
Comment (borfast) [reply site/pt.njk]: Indeed there is, well caught! Fixing it right away.

Comment (borfast) [reply site/pt.njk]: Done.

**Review by ahunts — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**